proto/gds/models/v1/models.proto

```diff
@@ -148,5 +148,5 @@ message EmailLogEntry {
 // See https://cloud.google.com/apis/design/design_patterns#list_pagination for more.
 message PageCursor {
     int32 page_size = 1;  // the number of results returned on each iteration.
-    bytes next_key = 2;   // the key to start the iteration from
+    string next_vasp = 2; // the VASP id to start the iteration from
```

Comment:
It's possible that in the future we could generalize things, but the store is set up right now with `DirectoryIterator` for VASPs and a `CertificateIterator` for CertificateRequests that are tied to type. I went in hoping it would be more generic, hence `PageCursor` instead of `DirectoryPageCursor` - but we'll have to do some refactoring to get there I think.